CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,19 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v62.0.0...main)
+
+ ## FxA Client
+
+### What's new ###
+- Send-tab metrics are recorded. A new function, `fxa_gather_telemetry` on the
+  account object (exposed as `account.gatherTelemetry()` to Kotlin) which
+  returns a string of JSON.
+
+  This JSON might grow to support non-sendtab telemetry in the future, but in
+  this change it will have:
```

Comment:
nit: our changelog entries are usually past-tense, this reads a little future-tense-ey; "in this change it contains" or similar may be clearer.